plugins/site-example/build.gradle

```diff
@@ -1,27 +0,0 @@
-/*
```

Comment:
I meant that in this doc: https://raw.githubusercontent.com/elastic/elasticsearch/master/docs/plugins/authors.asciidoc

There is a link to `https://github.com/elastic/elasticsearch/blob/master/plugins/site-example/build.gradle` which has been removed with this PR.